2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -224,40 +235,46 @@ public void disabledPeriodic()
   
   public void initialize()
   {
-    robotInitialized = true;
```

Comment:
I see how you've made tracking of the initialization states more fine-grained. This seems like a good idea. Can we remove `robotInitialized` entirely, and call `initialize()` unconditionally in `autonomousInit` and `teleopInit`?

Reply:
agreed and done